src/components/widgets/app-progress/app-progress.component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import Dialog from '@material-ui/core/Dialog';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const AppProfile = ({ isAppWaiting }) => (
+  <Dialog
+    aria-labelledby="simple-dialog-title"
+    open={isAppWaiting}
+  >
```

Comment:
The dialog should held the props for block the background click and the escape button press.